pkgs/data/fonts/intel-one-mono/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenvNoCC, fetchurl, unzip}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "intel-one-mono";
+  version = "1.2.0";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/otf.zip";
+      sha256 = "sha256-VnXIaW77dRXvXB1Vr01xRQDMECltwzF/RMqGgAWnu5M=";
+     })
+    (fetchurl {
+      url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/ttf.zip";
+      sha256 = "sha256-kaz0DePeO8nvKVonYJhs5f2+ps/5Xo5psjg1hoxzaiU=";
+    })
+  ];
+
+  # Multiple packages are download in srcs folder
+  # -> need to specify where the unpack phase will `cd` into
+  # use the `srcs` folder as the Root of the unpack phase
+  sourceRoot = ".";
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  doCheck = false;
+  dontFixup = true;
```

Comment:
```suggestion
```
which of those are no-ops?

Reply:
what do you mean with no-ops?

Reply:
configurePhase and buildPhase should be doing nothing if the files they detect are missing. It is not necessary to disable those if they don't cause issues and it rather causes more problems and unexpected problem when doing overlays.

Reply:
I tested the derivation without that options and it still works. Removed.